terraform/container-registry.tf

```diff
@@ -0,0 +1,64 @@
+resource "aws_ecs_cluster" "data_refinery" {
+  name = "data-refinery"
+}
+
+resource "aws_iam_role" "ecs_service_role" {
+  name = "data-refinery-ecs-service"
+
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
```

Comment:
Where is this "Version" date coming from? Why 2012-10-17? It might be good to put that in the docs as well.